### DISCUSSION TITLE
Add node from MenuAction (no ContainerLoader available)

### DISCUSSION THREAD

**Discussion by freddupont** [gephi/gephi-plugins#256] [Q&A]
hello

I try to 
generate a new graph and add nodeand link to from a wizard launched from a menu action. however ther is no ContainerLoader available how can I do it do you have code sample for example?

regards

**Comment by mbastian** [comment]
Hi, if you want to add elements to the graph you can directly use the `GraphAPI`. The `ContainerLoader` belongs to the `ImportAPI`. The `GraphAPI` gives you direct edit access to the graph structure.
